### ISSUE TITLE
Improve "Add Server" error when SSL certs are not configured properly

### ISSUE THREAD

**Issue by koverholt** [rstudio/rsconnect-jupyter#169]
If you attempt to add a connection to an RSC server that has self-signed certs or certs that are otherwise untrusted by the Python env (which can happen separately from certs trusted by the system), the error is fairly generic:

```
Failed to verify RSConnect Connect is running at
https://ec2-100-25-44-204.compute-1.amazonaws.com/.
Please ensure the server address is valid.
```

<img width="789" alt="Screen Shot 2019-05-16 at 9 31 07 AM" src="https://user-images.githubusercontent.com/303236/57885128-e7b81200-77ef-11e9-8354-19a4517d28fd.png">

Could this error be improved to indicate an issue with the SSL certificate? Or pass along the underlying error?

For comparison, push-button publishing in the IDE indicates a more specific cert error:

<img width="631" alt="Screen Shot 2019-05-16 at 3 34 35 PM" src="https://user-images.githubusercontent.com/303236/57885243-1e8e2800-77f0-11e9-8c37-61bb775f96e8.png">

**Comment by koverholt** [comment]
Note that we added a section to the following support article for users running into this issue:

[Troubleshooting Python with RStudio Connect](https://support.rstudio.com/hc/en-us/articles/360018735914).

**Comment by mmarchetti** [comment]
Fixed in #184
